src/convertToObject.js

```diff
@@ -3,16 +3,28 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
+
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const ROW_SPLITTER = ';\n';
+  const KEY_VALUE_SPLITTER = ':';
+  // const formatedStyle = {};
+  const removedEmptySpaces = sourceString
+    .split(ROW_SPLITTER)
+    .filter((string) => string.includes(KEY_VALUE_SPLITTER));
```

Comment:
I think we can remove this `filter` and in our reduce check if value !== undefined